puppetlabs_spec_helper.gemspec

```diff
@@ -26,12 +26,14 @@ Gem::Specification.new do |spec|
   spec.executables = Dir['bin/**/*'].map { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.required_ruby_version = Gem::Requirement.new('>= 2.7'.freeze)
+  spec.required_ruby_version = Gem::Requirement.new('>= 2.7')
 
   spec.add_runtime_dependency 'mocha', '~> 1.0'
   spec.add_runtime_dependency 'pathspec', '~> 1.0'
   spec.add_runtime_dependency 'puppet-lint', '~> 3.0'
   spec.add_runtime_dependency 'puppet-syntax', '~> 3.0'
   spec.add_runtime_dependency 'rspec-github', '~> 2.0'
   spec.add_runtime_dependency 'rspec-puppet', '~> 2.0'
+
+  spec.requirements << 'puppet, >= 6.0.0'
```

Comment:
Late to the party, but AFAIK Puppet 6 doesn't run on Ruby 2.7. I'd expect either Ruby 2.5 + Puppet 6 or Ruby 2.7 + Puppet 7.